designsafe/static/scripts/workspace/controllers/data-browser.js

```diff
@@ -1,4 +1,5 @@
 import _ from 'underscore';
+import { FieldReconPublishedViewComponent } from '../../data-depot/components/published/published-view.component';
```

Comment:
Is this import necessary?

Reply:
Weird, I have no idea how this line got here. I'll remove it.